website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -54,3 +67,61 @@ Options:
+`FIREZONE_DNS_CONTROL` environment variable to `systemd-resolved` or
+`etc-resolv-conf`. If you don't set this variable, Split DNS will be disabled.
+
+When Split DNS is enabled, the Linux Client will override the system's DNS
+resolver with its internal proxy. To do this, the Linux client needs to know how
+DNS resolution is currently being handled on your system.
+
+#### systemd-resolved
+
+On most modern Linux distributions, DNS resolution is handled by
+`systemd-resolved`. If this is the case for you, set `FIREZONE_DNS_CONTROL` to
+`systemd-resolved`. If you're not sure, you can check by running the following
+command:
+
+```bash
+systemctl status systemd-resolved
+```
+
+#### NetworkManager
+
+In most cases, if you're using NetworkManager your system is likely already
+using `systemd-resolved`, and you should set `FIREZONE_DNS_CONTROL` to
+`systemd-resolved`. You'll need to ensure that `/etc/resolv.conf` is a symlink
+to `/run/systemd/resolve/stub-resolv.conf`:
+
+```bash
+sudo ln -sf /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf
+```
+
+When NetworkManager detects that symlink exists, it will automatically use
+`systemd-resolved` for DNS resolution.
+
+#### `/etc/resolv.conf`
+
+If you're not using `systemd-resolved`, Firezone supports using the
+`/etc/resolv.conf` file to configure Split DNS as a fallback. To do this, set
+`FIREZONE_DNS_CONTROL` to `etc-resolv-conf`, and the Linux Client will override
+the `/etc/resolv.conf` file with the Firezone internal proxy.
+
+[Read more](/kb/deploy/dns) about how DNS works in Firezone.
+
+### Environment variable reference
+
+Here's an exhaustive list of environment variables you can use for the Linux
+client.
+
+| Variable Name          | Default Value       | Description                                                                                                                                                                                                  |
+| ---------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `FIREZONE_TOKEN`       |                     | Service account token generated by the portal to authenticate this client.                                                                                                                                   |
+| `FIREZONE_NAME`        | `<system hostname>` | Friendly name for this client to display in the UI.                                                                                                                                                          |
+| `FIREZONE_ID`          | `<random uuidv4>`   | Identifier used by the portal to identify this client for metadata and display purposes.                                                                                                                     |
```

Comment:
Do we make this random by default or was that in an issue / PR somewhere?

Reply:
Whoops, I thought this was the case. We have it for the gateway. See #3815 

I was gonna take a stab at it, but probably so much faster for you or @conectado to knock out